src/wd.rs

```diff
@@ -7,9 +7,38 @@ use serde::{Deserialize, Serialize};
 use std::borrow::Cow;
 use std::convert::TryFrom;
 use std::fmt;
+use std::fmt::Debug;
 use std::time::Duration;
 use webdriver::command::TimeoutsParameters;
 
+/// A command that can be sent to the WebDriver.
+///
+/// Anything that implements this command can be sent to [`Client::issue_cmd()`] in order
+/// to send custom commands to the WebDriver instance.
+pub trait WebDriverCompatibleCommand: Debug {
```

Comment:
We should add a blanket `impl` of this for `&T` and `Box<T>` where `T: WebDriverCompatibleCommand`. Will make it a little nicer to work with.